vendor/qtidataservices_app.te

```diff
@@ -0,0 +1,20 @@
+type qtidataservices_app, domain;
+app_domain(qtidataservices_app)
+net_domain(qtidataservices_app)
+
+get_prop(qtidataservices_app, vendor_default_prop)
+set_prop(qtidataservices_app, exported_radio_prop)
```

Comment:
What particular properties are being accessed here?

Reply:
`export_radio_prop` IIRC is necessary to make it run as `_app` and not `radio`, `vendor_default_prop` is used by `IWlanNetworkSer` and `NetworkService`.